lib/src/session-replay-mobile.ts

```diff
@@ -2,6 +2,13 @@ import type * as SessionReplay from '../generated/mobileSessionReplay'
 
 export * from '../generated/mobileSessionReplay'
 
+export const MobileSource: SessionReplay.MobileSegmentMetadata['source'][] = [
+  'android',
+  'ios',
+  'flutter',
+  'react-native',
+]
+
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Did you consider using the same pattern as other "enum"-like structures?

```suggestion
export const MobileSource = {
  Android: 'android',
  Ios: 'ios',
  Flutter: 'flutter',
  ReactNative: 'react-native'
} as const
export type MobileSource = typeof MobileSource[keyof typeof MobileSource]
```

Reply:
Seen on Slack: there is no point in using an object as those values will exclusively be used as an array.

Reply:
Eager to hear @mquentin 's take